src/tests/CartPageTest.java

```diff
@@ -0,0 +1,12 @@
+package tests;
+
+public class CartPageTest {
+	
+		
+	public void cartTest() {
+		
+		System.out.println("true");
```

Comment:
Hi Janani,

where is order page actions

Reply:
Done. Logic action added.